lib/ruby_indexer/lib/ruby_indexer/configuration.rb

```diff
@@ -44,10 +41,21 @@ def initialize
       if path
         # Substitute Windows backslashes into forward slashes, which are used in glob patterns
         glob = path.gsub(/[\\]+/, "/")
-        @excluded_patterns << File.join(glob, "**", "*.rb")
+        glob.delete_suffix!("/")
+        @excluded_patterns << "#{glob}/**/*.rb"
       end
 
-      @included_patterns = T.let([File.join("**", "*.rb")], T::Array[String])
+      excluded_directories = ["tmp", "node_modules", "sorbet"]
+      top_level_directories = Dir.glob("#{Dir.pwd}/*").filter_map do |path|
```

Comment:
We could extract `top_level_directories` to a method to avoid making `initializer` longer.